Description: Do not build planets-qml example
 The example fails to build from source on 32 bits archs, possibly requiring
 qt_add-big_resources instead of qt_add_resources. While trying to do that
 I also realized that upstream removed the example in Qt 6.5, so it is not
 worth the time to fix it.
Forwarded: not-needed
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>

---
 examples/qt3d/CMakeLists.txt |    1 -
 1 file changed, 1 deletion(-)

--- a/examples/qt3d/CMakeLists.txt
+++ b/examples/qt3d/CMakeLists.txt
@@ -13,7 +13,6 @@ if(TARGET Qt::Quick)
     qt_internal_add_example(controlsunderlay)
     qt_internal_add_example(instanced-arrays-qml)
     qt_internal_add_example(multiviewport)
-    qt_internal_add_example(planets-qml)
 #    qt_internal_add_example(scene2d) # Restore once Scene2D support has been ported to Qt6
     qt_internal_add_example(scene3d)
     qt_internal_add_example(simple-qml)
